weblab_cg/templates/chaste/normal_model.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef {% filter upper %}{{class_name}}FROMCELLML_HPP_{% endfilter %}
+#define {% filter upper %}{{class_name}}FROMCELLML_HPP_{% endfilter %}
+
+//! @file
+//! 
+//! This source file was generated from CellML.
+//! 
+//! Model: {{model_name_from_file}}
+//! 
+//! Processed by webalab_cg - CellML Tools in Python
```

Comment:
There's lots of CellML tools in python!

Maybe we could put a link to the repo or something?

Reply:
good idea!